### PR TITLE
Shows nearest busses

### PR DESCRIPTION
**Pull request by kwunkiuma**

### CODE REVIEW

**Comment by kwunkiuma** [comment]
`var resource =
                $"StopPoint?stopTypes=NaptanPublicBusCoachTram&radius=400&useStopPointHierarchy=false&modes=bus&categories=none&returnLines=false&lat={postcodeEntry.result.latitude}&lon={postcodeEntry.result.longitude}";
`
Is there a nice way to make long strings like this more readable?

File: BusBoard.ConsoleApp/ApiHelper.cs [hunk 1]
```diff
@@ -13,5 +14,30 @@ public static IEnumerable<T> ApiGet<T>(string baseUrl, string resource)
...
+
+        public static PostcodeEntry GetPostcodeEntry(string postcode)
+        {
+            return ApiGet<PostcodeEntry>("https://api.postcodes.io/", $"postcodes/{postcode}").First();
+        }
+
+        public static IEnumerable<StopPointEntry.StopPoint> GetNearestStopPoints(PostcodeEntry postcodeEntry)
+        {
+            var resource =
+                $"StopPoint?stopTypes=NaptanPublicBusCoachTram&radius=400&useStopPointHierarchy=false&modes=bus&categories=none&returnLines=false&lat={postcodeEntry.result.latitude}&lon={postcodeEntry.result.longitude}";
```
Comment (FridaTveit) [inline BusBoard.ConsoleApp/ApiHelper.cs]: You can split the string over multiple lines: https://stackoverflow.com/questions/1100260/multiline-string-literal-in-c-sharp 🙂

File: BusBoard.ConsoleApp/ApiHelper.cs [hunk 2]
```diff
@@ -13,5 +14,30 @@ public static IEnumerable<T> ApiGet<T>(string baseUrl, string resource)
...
+
+            return ApiGet<StopPointEntry>("https://api.tfl.gov.uk/", resource)
+                .First()
```
Comment (FridaTveit) [inline BusBoard.ConsoleApp/ApiHelper.cs:35]: Can this handle no stop points being returned?

File: BusBoard.ConsoleApp/RadiusEntry.cs [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+            public List<string> lineIdentifier { get; set; }
+        }
+
+        public class LineModeGroup
+        {
+            public string modeName { get; set; }
+            public List<string> lineIdentifier { get; set; }
+        }
+
+        public class AdditionalProperty
```
Comment (FridaTveit) [inline BusBoard.ConsoleApp/RadiusEntry.cs]: More descriptive name? 🙂

File: BusBoard.ConsoleApp/RadiusEntry.cs [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        {
+            public string category { get; set; }
+            public string key { get; set; }
+            public string sourceSystemKey { get; set; }
+            public string value { get; set; }
+            public DateTime modified { get; set; }
+        }
+
+        public class AdditionalProperty2
```
Comment (FridaTveit) [inline BusBoard.ConsoleApp/RadiusEntry.cs]: More descriptive name please! 🙂 Same for the two classes below

File: BusBoard.ConsoleApp/RadiusEntry.cs [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusBoard.ConsoleApp
+{
+    class RadiusEntry
+    {
+        public class PassengerFlow
```
Comment (FridaTveit) [inline BusBoard.ConsoleApp/RadiusEntry.cs]: No nested classes please! 🙂

File: BusBoard.ConsoleApp/StopPointEntry.cs [hunk 6]
```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BusBoard.ConsoleApp
+{
+    class StopPointEntry
```
Comment (FridaTveit) [inline BusBoard.ConsoleApp/StopPointEntry.cs]: Is this the same as the class above?

File: BusBoard.ConsoleApp/ApiHelper.cs [hunk 7]
```diff
@@ -13,5 +14,30 @@ public static IEnumerable<T> ApiGet<T>(string baseUrl, string resource)
...
+                .stopPoints;
+        }
+
+        public static string GetArrivingBusses(StopPointEntry.StopPoint stopPoint)
```
Comment (FridaTveit) [inline BusBoard.ConsoleApp/ApiHelper.cs]: One s in buses I think? 🙂

**Review by FridaTveit — COMMENTED** [review]
A few comments, otherwise looks good 🙂

File: BusBoard.ConsoleApp/ApiHelper.cs (hunk 2 above)
Comment (kwunkiuma) [reply BusBoard.ConsoleApp/ApiHelper.cs:35]: Yes it can, but should I add a conditional access?

Comment (FridaTveit) [reply BusBoard.ConsoleApp/ApiHelper.cs:35]: So what currently happens if the api returns no stop points? 🙂

Comment (kwunkiuma) [reply BusBoard.ConsoleApp/ApiHelper.cs:35]: It returns an empty list and the foreach loop after it skips past.

Comment (FridaTveit) [reply BusBoard.ConsoleApp/ApiHelper.cs:35]: I see, that sounds fine 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**